config/registry/aws/modules/aws-k8s-service.yaml

```diff
@@ -162,6 +162,12 @@ inputs:
       A list persistent storages to add to each instance of your service (need to give a `size` which is the size in GB
       for the storage volume to be, and `path` which is the path in the filesystem of each instance to place it under)
     default: []
+  - name: annotations
```

Comment:
I like `ingress_annotations` or `ingress_extra_annotations` (since you adding annotations, not setting all of them).

Reply:
In the PR the key for ingress annotations for k8s-service is 

annotations-->ingress-->[ list of kv pairs for annotations ]

The 2 levels are because annotations may be needed for other objects in the k8s-service chart in the future. 

How about

annotations --> ingress_extra_annotations -->[ list of kv pairs for annotations ]

?

But if you both @NitinAgg @rabbitfang  think the single layer is adequate, lets just do that and continue.

Reply:
Yes, let's do `ingress_extra_annotation` mainly because otherwise the user would have to somehow figure out what are the possible keys inside `annotations`. And it is just be cleaner to put all those potential keys at top level

Reply:
Ok, I've made the changes as per your suggestion.